rs/bitcoin/adapter/tests/adapter_test.rs

```diff
@@ -158,6 +159,7 @@ fn start_adapter_and_client(
     urls: Vec<SocketAddr>,
     logger: ReplicaLogger,
     network: bitcoin::Network,
+    is_idle: bool,
```

Comment:
Instead of using boolean flags, you can use an enum instead. It makes it easier to read when looking at the call sites, and then you don't need the two wrapper functions `start_idle` and `start_active`.

You can do something like:
```rust
enum AdapterState {
 Active,
 Idle,
}
```

```suggestion
    is_idle: AdapterState,
```

Reply:
done. thanks!